ingest/build.gradle

```diff
@@ -13,5 +13,11 @@ docker {
 	buildArgs([JAR_FILE: "${tasks.bootJar.outputs.files.singleFile.name}"])
 }
 
+dependencies {
+	implementation platform("software.amazon.awssdk:bom:2.5.29")
+	implementation "software.amazon.awssdk:s3"
+
```

Comment:
What's the difference between `platform("x")` and `x`?

Reply:
Maven's BOM(Bill of Materials) is a kind of platform that is used to keep the sdk versions up to date.